client/src/Movies/Movie.js

```diff
@@ -25,17 +26,18 @@ export default class Movie extends Component {
         console.error(error);
       });
   };
+  
   // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+  componentWillReceiveProps(newProps){
```

Comment:
I know this method was provided in the assignment, but it's deprecated, let's use CDU